server/Arcadia.Assistant.Organization.Abstractions/EmployeeMetadata.cs

```diff
@@ -1,26 +1,26 @@
 ﻿namespace Arcadia.Assistant.Organization.Abstractions
 {
     using System;
+    using System.Collections.Generic;
     using System.Diagnostics;
 
     public class EmployeeMetadata
     {
-        public EmployeeMetadata(string employeeId, string name, string identity, string email)
+        public EmployeeMetadata(string employeeId, string name, string email)
         {
             this.EmployeeId = employeeId;
             this.Name = name;
-            this.Identity = identity;
             this.Email = email;
         }
 
         public string EmployeeId { get; }
 
         public string Name { get; set; }
 
-        public string Identity { get; set; }
-
         public string Email { get; set; }
 
+        public IEnumerable<string> IdentityAliases { get; set; }
```

Comment:
default value maybe?

Reply:
I made such at first, but anyway it can be set to null and we need null-propagation, so why do we need that default value?

Reply:
i would not use null propagation, that's all :)

Reply:
basically, I'd try to avoid nulls if possible. '?' makes it safe but uncertain

Reply:
Everyone needs not-null reference values in C# then :)

Reply:
Done.